algorithms/kernel/low_order_moments/low_order_moments_estimates_online.i

```diff
@@ -283,17 +283,20 @@ Status compute_estimates(NumericTable * dataTable, PartialResult * partialResult
     const size_t numRowsBlocks      = _cd.nVectors / numRowsInBlock;
     const size_t numRowsInLastBlock = numRowsInBlock + (_cd.nVectors - numRowsBlocks * numRowsInBlock);
 
+    DAAL_ITTNOTIFY_SCOPED_TASK(LowOrderMomentsOnlineTask.compute);
```

Comment:
Do you really need this task?

Reply:
Yes, it aggregates all activities related to compute, and added here for aligning with GPU branch.

Reply:
Maybe it makes sense to remove from the GPU?